tests/integrations/asgi/test_asgi.py

```diff
@@ -202,3 +206,42 @@ def handler(*args, **kwargs):
     (exception,) = event["exception"]["values"]
     assert exception["type"] == "ValueError"
     assert exception["value"] == "oh no"
+
+async def test_traces_sampler_gets_scope_object_in_sampling_context(
+        sentry_init,
+        app,
+        capture_events,
+        DictionaryContaining,
+        ObjectDescribedBy
+):
+    test_url = "/sync-message"
+
+    traces_sampler = mock.Mock()
+    sentry_init(
+        send_default_pii=True,
+        traces_sampler=traces_sampler
+    )
+
+    events = capture_events()
```

Comment:
can you do something with this var, such as asserting the event did/did not get sent?